doc/qbk/4.1.parsing.qbk

```diff
@@ -12,14 +12,46 @@
 [#section.url_view]
 [section Parsing]
 
-[#url_notation]
-[heading Notation]
+All parsing functions accept a __string_view__ and return a
+`__result__<__url_view__>`. The following example parses a string literal
```

Comment:
"Algorithms which parse URLs return a view which references the underlying character buffer without taking ownership, avoiding memory allocations and copies. However, the caller is responsible for ensuring that the lifetime of the character buffer extends until it is no longer referenced by the view."